### PR TITLE
Cu 82pdw3 css integration badge

### PR DESCRIPTION
**Pull request by Hamza-A**
This branch aims to add new badges "primary" and "awaiting validation", stylized as on the wireframe to identity's page

## Description
To do this, I've created new components and their stories:
- `<PrimaryBadge />` which contains a `<p>` tag and a `<PrimaryIcon />`
- `<AwaitingValidationBadge />` which contains a `<p>` and a `<AwaitingValidationIcon />`
Those components has been implemented in `<identityLine />`

## How Has This Been Tested?
There was some changes to add on test files, I'm now using the components I've created in tests.

## Screenshots
image
image

## Types of changes

- Chore (non-breaking change which refactors / improves the existing code base)

## Checklist:

- :green_circle: My code follows the code style of this project.
- :red_circle: My change requires a change to the documentation.
- :red_circle: I have updated the documentation accordingly.
- :green_circle: I have read the [**CONTRIBUTING**][contributing_file] document.
- :red_circle: I have added tests to cover my changes.
- :green_circle: All new and existing tests passed.

[contributing_file]: https://github.com/fewlinesco/connect/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: src/components/IdentityLine.tsx [hunk 1]
```diff
@@ -20,14 +24,22 @@ export const IdentityLine: React.FC<IdentityLineProps> = ({ identity }) => {
         <Flex>
           <Value>{value}</Value>
         </Flex>
-        {primary && status === "validated" && <p>Primary</p>}
+        {primary && status === "validated" && (
+          <PrimaryBadge>
+            <p>Primary</p>
+            <PrimaryIcon />
+          </PrimaryBadge>
```
Comment (Fuzznimp) [inline src/components/IdentityLine.tsx:31]: This should be inside the `PrimaryBadge` (i.e. here, it should be `<PrimaryBadge/>`).

File: src/components/IdentityLine.tsx [hunk 2]
```diff
@@ -20,14 +24,22 @@ export const IdentityLine: React.FC<IdentityLineProps> = ({ identity }) => {
...
           <IdentityInfo>
             <p>Added on ...</p>
             <p>Last used to login on ...</p>
           </IdentityInfo>
         ) : (
-          <p>awaiting validation</p>
+          <AwaitingValidationBadge>
+            <p>awaiting validation</p>
+            <AwaitingValidationIcon />
+          </AwaitingValidationBadge>
```
Comment (Fuzznimp) [inline src/components/IdentityLine.tsx:42]: Same as PrimaryBadge.

File: src/components/display/fewlines/AwaitingValidationIcon/AwaitingValidationIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      />
+      <path
+        d="M7.00004 10.3333C7.36823 10.3333 7.66671 10.0349 7.66671 9.66667C7.66671 9.29848 7.36823 9 7.00004 9C6.63185 9 6.33337 9.29848 6.33337 9.66667C6.33337 10.0349 6.63185 10.3333 7.00004 10.3333Z"
+        fill="white"
+      />
+      <path
+        d="M7.00004 3.6665C6.82323 3.6665 6.65366 3.73674 6.52864 3.86177C6.40361 3.98679 6.33337 4.15636 6.33337 4.33317V7.6665C6.33337 7.84331 6.40361 8.01288 6.52864 8.13791C6.65366 8.26293 6.82323 8.33317 7.00004 8.33317C7.17685 8.33317 7.34642 8.26293 7.47144 8.13791C7.59647 8.01288 7.66671 7.84331 7.66671 7.6665V4.33317C7.66671 4.15636 7.59647 3.98679 7.47144 3.86177C7.34642 3.73674 7.17685 3.6665 7.00004 3.6665Z"
+        fill="white"
+      />
+    </svg>
```
Comment (Fuzznimp) [inline src/components/display/fewlines/AwaitingValidationIcon/AwaitingValidationIcon.tsx:24]: I have you used `npx svgo` on the 2 svgs?

**Review by Fuzznimp — CHANGES_REQUESTED** [review]

File: src/components/IdentityLine.tsx (hunk 1 above)
Comment (Hamza-A) [reply src/components/IdentityLine.tsx]: OK fixed 6ed6a28

File: src/components/IdentityLine.tsx (hunk 2 above)
Comment (Hamza-A) [reply src/components/IdentityLine.tsx]: OK fixed 6ed6a28

File: src/components/display/fewlines/AwaitingValidationIcon/AwaitingValidationIcon.tsx (hunk 3 above)
Comment (Hamza-A) [reply src/components/display/fewlines/AwaitingValidationIcon/AwaitingValidationIcon.tsx]: No I didn't, now I did 11b5261

Comment (Fuzznimp) [reply src/components/display/fewlines/AwaitingValidationIcon/AwaitingValidationIcon.tsx]: Haha, see how effective it is? And how svgs can be bloated haha.

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**